rl4co/envs/graph/__init__.py

```diff
@@ -0,0 +1,3 @@
+from rl4co.envs.graph.mcp.env import MCPEnv
+from rl4co.envs.graph.flp.env import FLPEnv
```

Comment:
[Minor] We should add these envs to the `rl4co.envs.__init__.py` so they are importable from there too

Reply:
Added! Thanks!